src/constants.ts

```diff
@@ -1,6 +1,35 @@
-export const DEBOUNCE_DELAY = 175;
+const REPEATABLE_SYMBOLS = [
+  "\\+",
+  "-",
+  "\\*",
+  "/",
+  "=",
+  "<",
+  ">",
+  "_",
+  "#",
+  "\\.",
+  "\\|",
+  "&"
+].map(s => `${s}+`).join("|");
```

Comment:
Let's leave it for now, but def starting to feel a bit complex for `constants.py` 😅

Reply:
Yea that is true.
I'm running it locally now as well.